lib/widgets/emoji_reaction.dart

```diff
@@ -417,20 +417,21 @@ void showEmojiPickerSheet({
     // on my iPhone 13 Pro but is marked as "much slower":
     //   https://api.flutter.dev/flutter/dart-ui/Clip.html
     clipBehavior: Clip.antiAlias,
+    // This removes insets from all sides except bottom.
+    // Leave it to the descendent [CustomScrollView].
```

Comment:
I don't know what "removes insets" means. Taken literally, it doesn't make sense; the insets are features of the device (which might include hardware) that our app isn't able to remove.

How about:

```dart
    // The bottom inset is left for [builder] to handle;
    // see [EmojiPicker] and its [CustomScrollView] for how we do that.
```

It's also possible that we don't need a comment at all, and the `useSafeArea` dartdoc is sufficient if readers are curious.